src/main/java/net/neoforged/neoforge/transfer/transaction/SnapshotJournalDebugInfo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.transfer.transaction;
+
+import it.unimi.dsi.fastutil.objects.Object2IntMap;
+import it.unimi.dsi.fastutil.objects.Object2IntMaps;
+import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
+import net.neoforged.neoforge.transfer.ResourceHandlerDeprecationHandling;
+import org.jetbrains.annotations.ApiStatus;
+
+// TODO: Remove after migrations have been established. This is more for info to validate assumptions on use rather than guessing and inquiring a subset.
+@Deprecated(since = ResourceHandlerDeprecationHandling.MC_1_21_6)
+@ApiStatus.Internal
+public class SnapshotJournalDebugInfo {
```

Comment:
Why is this still here? I don't understand what you expect to gain from this information. Additionally, it is not thread-safe.